packages/core/content-releases/server/src/destroy.ts

```diff
@@ -0,0 +1,17 @@
+import { Job } from 'node-schedule';
+import { LoadedStrapi } from '@strapi/types';
+
+import { Release } from '../../shared/contracts/releases';
+import { getService } from './utils';
+
+export const destroy = async ({ strapi }: { strapi: LoadedStrapi }) => {
+  if (strapi.features.future.isEnabled('contentReleasesScheduling')) {
+    const scheduledJobs: Map<Release['id'], Job> = getService('scheduling', {
+      strapi,
+    }).syncFromDatabase();
```

Comment:
```suggestion
    const scheduledJobs: Map<Release['id'], Job> = await getService('scheduling', {
      strapi,
    }).syncFromDatabase();
```

Reply:
why the destroy has to sync from database? Wouldn't it be quicker to access the scheduled jobs at that point without the db query?

Reply:
This is definitely a mistake, I wanted to use getAll there 😅